test/cmocka/CMakeLists.txt

```diff
@@ -1,46 +1,42 @@
 project(obs-cmocka)
 
-# Fix libobs path
-macro(fixLink target_arg)
-	if(APPLE AND UNIX)
-		add_custom_command (TARGET ${target_arg}
-			POST_BUILD COMMAND "${CMAKE_INSTALL_NAME_TOOL}"
-			"-change" "@rpath/libobs.0.dylib" "@executable_path/../../libobs/libobs.0.dylib"
-			"$<TARGET_FILE:${target_arg}>" VERBATIM)
-	endif()
-endmacro()
-
-set(CMAKE_MACOSX_RPATH TRUE)
-set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
-list(APPEND CMAKE_INSTALL_RPATH "@loader_path/" "@executable_path/")
-
 find_package(CMocka CONFIG REQUIRED)
-include_directories(${CMOCKA_INCLUDE_DIR})
-include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/libobs")
-
-# fix rpath on linux
-if (UNIX AND NOT APPLE)
-	set(CMAKE_INSTALL_RPATH "$ORIGIN";../../libobs)
-endif()
 
 # Serializer test
 add_executable(test_serializer test_serializer.c)
-target_link_libraries(test_serializer ${CMOCKA_LIBRARIES} libobs)
+target_include_directories(test_serializer PRIVATE ${CMOCKA_INCLUDE_DIR})
+target_link_libraries(test_serializer PRIVATE OBS::libobs ${CMOCKA_LIBRARIES})
 
 add_test(test_serializer ${CMAKE_CURRENT_BINARY_DIR}/test_serializer)
-fixLink(test_serializer)
-
+# fixLink(test_serializer)
```

Comment:
If this isn't needed, it can just be removed.

Reply:
This change (and the others in the test sub-project) I commented out for now, because I'd like @DDRBoxman to check if the tests still work as intended (iirc they added them originally?).